src/components/Explore/utils.ts

```diff
@@ -430,6 +435,21 @@ export const exploreMetricData: {
       },
     ],
   },
+  [ExploreMetric.WEEKLY_DEATHS_PER_100K]: {
+    title: 'Weekly reported deaths per 100k',
+    name: 'Weekly reported deaths per 100k',
```

Comment:
Same feedback here, these shouldn't be "per 100k" by default...

That said, ideally when you added 2+ locations to the chart, then it would switch to "per 100k" like we do for cases now:
![image](https://user-images.githubusercontent.com/206364/163067640-92cb63b9-a335-40d3-8354-da907abacd26.png)

![image](https://user-images.githubusercontent.com/206364/163067681-e6c9c556-5be7-422a-805e-5c9cd00ccb91.png)

I'm not sure how this logic works, but I think we would duplicate it for "weekly cases" and "weekly deaths".  When you add a second location, the data would be normalized by population and the name of the metric would change to include "per 100k".

Reply:
@faihegberg this logic happens by adding the metric [to this array](https://github.com/covid-projections/covid-projections/blob/15ea16c9fa841cd3a8be1ea606a4c51e7ea43e7f/src/components/Explore/utils.ts#L108-L118).

you can add `ExploreMetric.WEEKLY_CASES_PER_100K` to this too (in addition to `ExploreMetric.WEEKLY_DEATHS_PER_100K`)

after adding both metrics, you can remove the "per 100k" from the `name` and `title` for both in [exploreMetricData](https://github.com/covid-projections/covid-projections/blob/15ea16c9fa841cd3a8be1ea606a4c51e7ea43e7f/src/components/Explore/utils.ts#L227-L229), as well as the `label` and `tooltipLabel` in each `seriesList` array